pkg/storaged/vdos-panel.jsx

```diff
@@ -58,7 +58,7 @@ export class VDOsPanel extends React.Component {
                                 {
                                     empty_warning: _("No disks are available."),
                                     validate: function (spc) {
-                                        if (spc === undefined)
+                                        if (spc === null)
```

Comment:
It's not at all obvious that this is correct -- why does it need to be so specific? If  this is some input value, then `if (!spc)` ought to be more robust and suffice?

Reply:
Yes, good point.  I can't recall how we ended up with the current code.